ee/docker-ee/ubuntu.md

```diff
@@ -119,19 +119,20 @@ and update Docker Engine - Enterprise from the repository.
 
 4. Temporarily add a `$DOCKER_EE_VERSION` variable into your environment.
 
-   > **Note:** If you need to run Docker Enterprise 2.0, please see the
-   > following instructions:
-   > * [18.09](https://docs.docker.com/v18.09/ee/supported-platforms/) - Docker
-   >   Enterprise Edition 2.1 (Docker Engine, UCP, and DTR).
-   > * [18.03](https://docs.docker.com/v18.03/ee/supported-platforms/) - Older
-   >   Docker Engine - Enterprise only release
-   > * [17.06](https://docs.docker.com/v17.06/engine/installation/) - Docker
-   >   Enterprise Edition 2.0 (Docker Engine, UCP, and DTR).
-
     ```bash
     $ DOCKER_EE_VERSION={{ site.docker_ee_version }}
     ```
-
+    > **Note** 
+    > 
+    > If you need to run Docker Enterprise 2.0, please see the
+    > following instructions:
```

Comment:
I suggest editing the sentence to read: "If you need to run Docker Enterprise 2.0, refer to the following:"